float64 in_temp
float64 out_temp
float64 in_humi
float64 out_humi
float64 wind_sp
float64 wind_dir
float64 press
float64 rain
float64 cabin_temp1
float64 cabin_temp2
float64 dome_temp1
float64 dome_temp2
float64 gen_temp1
float64 gen_temp2
string from_node
float64 timestamp

